### PR TITLE
fix: layout & paid button on overview screen

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: src/components/UsageDetails/index.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const shouldShowUpgrade = (plan, appCount) => {
 	if (plan === 'free') {
 		return { show: true };
 	}
-	if (plan === 'growth') {
+	if (plan === 'growth' || plan === 'bootstrap') {
```
Comment (bietkul) [inline src/components/UsageDetails/index.js:26]: @jyash97 We need to show the usage details for bootstrap users too.
Why did you set it to `false`?

**Review by bietkul — DISMISSED** [review]

**Comment by bietkul** [comment]
@siddharthlatest @jyash97 Why we don't have a netlify build for this PR?

File: src/components/UsageDetails/index.js (hunk 1 above)
Comment (siddharthlatest) [reply src/components/UsageDetails/index.js:26]: @bietkul This condition is only affecting whether to show the upgrade now button.

**Comment by siddharthlatest** [comment]
I am not sure, will check.

**Review by siddharthlatest — APPROVED** [review]
Tentatively approved (y)

**Pull request merged**